src/backend/distributed/planner/distributed_planner.c

```diff
@@ -309,11 +315,40 @@ distributed_planner(Query *parse, int cursorOptions, ParamListInfo boundParams)
 List *
 ExtractRangeTableEntryList(Query *query)
 {
-	List *rangeTblList = NIL;
+	List *rteList = NIL;
+
+	ExtractRangeTableEntryWalker((Node *) query, &rteList);
+
+	return rteList;
+}
+
+
+/*
+ * ExtractClassifiedRangeTableEntryList extracts reference table rte's from
+ * the given rte list.
+ * Callers of this function are responsible for passing referenceTableRTEList
+ * to be non-null and initially pointing to an empty list.
+ */
+void
+ExtractReferenceTableRTEList(List *rteList, List **referenceTableRTEList)
```

Comment:
Why not just return a value?

Reply:
Actually, this those commits are to be merged implementing single placement tables and there I am planning to add another parameter to this function to extract single placement tables as well.

But I can change the function signature here if it makes more sense

Reply:
Actually, those changes are part of ones to be merged before implementing single placement tables and there I am planning to add another parameter to this function to extract single placement tables as well.

But I can change the function signature here if it makes more sense

Reply:
done